docs/topics/proc_deploying-the-cache-booster-using-the-oc-cli-client.adoc

```diff
@@ -61,7 +62,7 @@ $ mvn clean fabric8:deploy -Popenshift
 ----
 endif::built-for-nodejs[]
 
-. Check the status of your booster and ensure your pod is running.
+. Check the status of your example and ensure your pod is running.
```

Comment:
Check the status of your example and ensure your pod is running. -->
Check the status of your application and ensure your pod is running.

Reply:
Thanks. Similarly, I made this change to the parallel modules.